DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/RemoveProcessor.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors
+{
+    /// <summary>
+    /// Used to remove the entire DICOM item.
+    /// </summary>
+    public class RemoveProcessor : IAnonymizerProcessor
+    {
+        public void Process(DicomDataset dicomDataset, DicomItem item, ProcessContext context = null)
+        {
+            EnsureArg.IsNotNull(dicomDataset, nameof(dicomDataset));
+            EnsureArg.IsNotNull(item, nameof(item));
+
+            dicomDataset.Remove(item.Tag);
+        }
+
+        public bool IsSupportedVR(DicomItem item)
```

Comment:
>DicomItem item [](http://example.com/codeflow?start=34&length=14)

C# don't generate a warning if the parameter is not used, right?

Reply:
I suppressed this warning.